.github/workflows/validate-submodule-pull-request.yml

```
@@ -15,10 +15,16 @@ on:
         required: false
         default: Lombiq/Open-Source-Orchard-Core-Extensions
         description: The parent repository's full name in "{owner}/{repository}" format.
+      timeout-minutes:
+        required: false
+        type: number
+        default: 10
```

```suggestion
        default: 3
```
10 min is too much.
See: https://github.com/Lombiq/Orchard-UIKit/actions/workflows/validate-pull-request.yml